test/functional/neutronless/bigip_interaction.py

```diff
@@ -231,31 +223,26 @@ def BEGIN():
     not execute, and all fixtures that use this library will not perform
     bigip config tracking.
 
-    WARNING: Nightly will perform this tracking; thus, a user does no justice
-    if linging config is later discovered.  It is; therefore, still recommended
-    to run this as a last step to assure proper config tracking is assured.
+    WARNING: Nightly will perform this tracking; thus, a user does themselves
+    no justice if lingering config is later discovered.  It is; therefore,
+    still recommended to run this as a last step to assure proper config
+    tracking is assured.
     """
     ssh_options = """ssh -o StrictHostKeyChecking=no \
     -o UserKnownHostsFile=/dev/null \
     {}"""
     ssh_host_specific_fmt = "{}@{}"
-    ssh_cmd = ''
     hostname = ''
     username = ''
-    if 'TESTENV_AGENT_BRANCH' in os.environ:
-        # we are in testenv...
-        hostname = os.environ['TESTENV_BIGIP_MGMT_IP_PUBLIC']
-        username = os.environ['TESTENV_BIGIP_SSH_USERNAME']
-    elif hasattr(pytest.symbols, 'bigip_mgmt_ip') and \
+    if hasattr(pytest.symbols, 'bigip_mgmt_ip') and \
             hasattr(pytest.symbols, 'bigip_ssh_username'):
         hostname = pytest.symbols.bigip_mgmt_ip
-        username = pytest.symbols.bigip_username
+        username = pytest.symbols.bigip_ssh_username
     else:
-        raise EnvironmentError("Cannot perform tests without symbols or "
-                               "expected testenv environment!")
+        raise EnvironmentError("Cannot perform tests without symbols!")
     ssh_cmd = ssh_options.format(
         ssh_host_specific_fmt.format(username, hostname))
     BigIpInteraction.ssh_cmd = ssh_cmd
 
 
-BEGIN()
```

Comment:
In the commit message:
"chanes" --> "changes"?

or just "Apply code review comments."?